### PR TITLE
[W5.4][T16-A2]LENALD NG WAI JUN 

### PR DESCRIPTION
**Pull request by Lenaldnwj**
Enhance find command so that it is case-insensitive.
Make changes to user guide.

### CODE REVIEW

**Comment by Kennard123661** [comment]
Hi Lenald, try amending your commit messages so it becomes more descriptive of your commits. https://help.github.com/articles/changing-a-commit-message/

File: src/seedu/addressbook/commands/FindCommand.java [hunk 1]
```diff
@@ -46,15 +43,34 @@ public CommandResult execute() {
      * @param keywords for searching
      * @return list of persons found
      */
-    private List<ReadOnlyPerson> getPersonsWithNameContainingAnyKeyword(Set<String> keywords) {
+    private List<ReadOnlyPerson> getPersonNameContainingSimilarPart(Set<String> keywords) {
```
Comment (brandonyeoxg) [inline src/seedu/addressbook/commands/FindCommand.java]: The header comment for this method could be updated to reflect the change in behaviour.

File: src/seedu/addressbook/commands/FindCommand.java [hunk 2]
```diff
@@ -46,15 +43,34 @@ public CommandResult execute() {
...
         return matchedPersons;
     }
 
+
+    /**
+     * Changes entire string to lower case
+     * @param set of string to be converted to lower case
+     * @return the strings in lower case.
+     */
+    private Set<String> nameToLowerCase(Set<String> set) {
```
Comment (brandonyeoxg) [inline src/seedu/addressbook/commands/FindCommand.java]: Does this method really convert a name to lowercase or a set of names? Perhaps the method name can be updated to reflect this.

File: src/seedu/addressbook/commands/FindCommand.java [hunk 3]
```diff
@@ -1,25 +1,22 @@
 package seedu.addressbook.commands;
 
-import java.util.ArrayList;
-import java.util.Collections;
-import java.util.HashSet;
-import java.util.List;
-import java.util.Set;
+import java.util.*;
```
[the remark this answers is not in the document]
Comment (Lenaldnwj) [reply src/seedu/addressbook/commands/FindCommand.java]: Ok noted, thank you!

File: src/seedu/addressbook/commands/FindCommand.java (hunk 1 above)
Comment (Lenaldnwj) [reply src/seedu/addressbook/commands/FindCommand.java]: Understood

**Comment by Lenaldnwj** [comment]
@brandonyeoxg I made changes to the user guide.

**Comment by brandonyeoxg** [comment]
@Lenaldnwj Hey Lenald, noted you fulfilled the LO requirements =). Please close this PR.

**Pull request closed without merging**